plugins/woocommerce/tests/php/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -75,13 +76,15 @@ private function initialize_merged_query( $parsed_block = array() ) {
 	/**
 	 * Build a simplified request for testing.
 	 *
-	 * @param bool  $woocommerce_on_sale WooCommerce on sale.
-	 * @param array $woocommerce_attributes WooCommerce attributes.
-	 * @param array $woocommerce_stock_status WooCommerce stock status.
+	 * @param string $featured WooCommerce featured.
+	 * @param bool   $woocommerce_on_sale WooCommerce on sale.
+	 * @param array  $woocommerce_attributes WooCommerce attributes.
+	 * @param array  $woocommerce_stock_status WooCommerce stock status.
 	 * @return WP_REST_Request
 	 */
-	private function build_request( $woocommerce_on_sale = 'false', $woocommerce_attributes = array(), $woocommerce_stock_status = array() ) {
+	private function build_request( $featured = 'false', $woocommerce_on_sale = 'false', $woocommerce_attributes = array(), $woocommerce_stock_status = array() ) {
```

Comment:
I didn't look too deeply into the whole class but would it not make sense to use an array here to set whatever terms that are passed in instead of just one specific term? What if we add more terms later, then we would need to change the shape of this method again.

Reply:
Well, you're not wrong :)